meshapp/src/profile/profile-page.tsx

```diff
@@ -51,7 +56,10 @@ const ProfilePage = (props: Profile) => {
   return (
     <Box className="profile-page-container">
       <Box className="profile-page-header">
-        <ProfileHeader name={props.name} pronouns={props.pronouns} />
+        
+        {/* <ProfileHeader name={props.name} pronouns={props.pronouns} /> Temporary Comment*/}
```

Comment:
What's this for?

Reply:
Old ProfileHeader Deleted. 
Renamed the class from ProfileName to ProfileHeader